charts/frontend/templates/configmap.yaml

```diff
@@ -239,7 +244,9 @@ data:
 
         ## Rather than just denying .ht* in the config, why not deny
         ## access to all .invisible files
+        {{- if (mergeOverwrite $.Values.serviceDefaults.nginx ($service.nginx)).denyDotFiles }}
```

Comment:
When I set `services.hello.nginx.denyDotFiles = false`, it seems to also affect the `world` service. Please check.